change/@microsoft-teams-js-f9182c21-8a37-405a-9fa4-9d394fece0dd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "1. Renamed filePath field to webkitRelativePath 2. Removed two validation checks for destinationFolder fields",
```

Comment:
References to code (like filePath, webkitRelativePath, etc.) should be surrounded by ``